lib/DataFileUtil/DataFileUtilClient.pm

```diff
@@ -0,0 +1,3352 @@
+package DataFileUtil::DataFileUtilClient;
```

Comment:
Right- I was going to switch, but it seemed to work without doing that.  That will be round 2.